### ISSUE TITLE
False negative in UnusedImportCheck

### ISSUE THREAD

**Issue by mahfouz72** [checkstyle/checkstyle#14697]
detected at https://github.com/checkstyle/checkstyle/pull/14548#discussion_r1529498536 and https://github.com/checkstyle/checkstyle/pull/14548#discussion_r1530815684
I have read check documentation: https://checkstyle.org/checks/xxxxxx/nameofaffectedcheck.html
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words
```

PS D:\test> cat src/Test.java                                                  


import java.util.List; // expected violation here but there is no violation

/**
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

 <module name="TreeWalker">

      <module name="UnusedImports"/>
    </module>
</module>
PS D:\test> java  -jar checkstyle-10.13.0-all.jar -c config.xml  src/Test2.java
Starting audit...
Audit done.
```

**Describe what you expect in detail.**

This is an invalid Link because there is no `{  }` so the check should Ignore this and It shouldn't be considered as usage So we should have a violation on `import java.util.List;`

**Comment by romani** [comment]
Please fix a link to check documentation in your PR description

**Comment by romani** [comment]
@zyadahmed , please help us to resolve this issue.
